tensordict/tensordict.py

```diff
@@ -3855,6 +3880,14 @@ def _update_valid_keys(self) -> None:
         self._valid_keys = sorted(valid_keys)
 
     def select(self, *keys: str, inplace: bool = False) -> LazyStackedTensorDict:
+        # TODO: Add support for nested keys.
+        for key in keys:
+            if not isinstance(key, str):
```

Comment:
I guess we'll need to make this work later, do you want to keep it for a second PR?

Reply:
Yes, I think let's leave it for a second PR, though I think that can come pretty quick once your changes to `LazyStackedTensorDict` get merged.